app/models/sample_type.rb

```diff
@@ -292,9 +305,5 @@ def validate_title_is_not_type_of_seek_sample_multi
     end
   end
 
-  def self.supports_extended_metadata?
-    false
-  end
-
```

Comment:
Just a question. Does it support Extended Metadata now? Or is it not necessary  anymore because it's an asset now?

Reply:
SampleTypes don't support EM. I didn't add this by the way, but just moved up the class out of the private area so I could group these overrides together